### DISCUSSION TITLE
ProxyJump: what am I doing wrong?

### DISCUSSION THREAD

**Discussion by avysk** [blinksh/blink#1840] [Support] [answered]
Hello,

I have the following setup:

1. There is a `remote` machine with SSH on port 22, which is unreachable from outside.
2. The `remote` machine is set to ssh to `bastion` and forward remote port 32322 to local 22.
3. If I am at `bastion`, then I can reach `remote` via `ssh -p 32322 localhost`.
4. The following setup works for Windows 11 client:
```
Host <remote>
    User alexey
    HostName localhost
    Port 32322
    ProxyJump <bastion>
```
and I can successfully `ssh <remote>`.
5. I have `<bastion>` set in Blink and `ssh <bastion>` works.
6. I try to set up `<remote>` in Blink by setting all the same values as in the config snippet above. My problem: `ssh <remote>` does not work; `ssh -vvv <remote>` prints `Executing proxycommand -W [localhost]:32322 <bastion>` and then does nothing, and if I enter this command manually I can see OpenSSH greeting from `<remote>` (it runs FreeBSD, and bastion runs Linux, so it is easy to say that the greeting indeed comes from `<remote>`).

What am I doing wrong?

**Comment by avysk** [comment]
I probably should add that
- Changing `localhost` to `127.0.0.1` does not help;
- using FQDN or pseudonym for `<bastion>` changes nothing;
- FreeBSD machine `<remote>` actually allows direct SSH from inside the home network, and I have Blink host set for it, and it works in the home network (so Blink does not have a problem with FreeBSD SSH).

**Reply by carloscabanero** [reply]
These are usually configuration issues. Could you send the .blink/ssh_config file? And also, the full output from -vvvvv? I will try to replicate from my side.

**Comment by avysk** [comment, marked as answer]
Forget about it. I was using TestFlight version; when I moved back to AppStore one a minute ago it started to work.
